### ISSUE TITLE
Update assets for UI

### ISSUE THREAD

**Issue by marvinmarnold** [NebraLtd/maker-starter-app#11]
Pictures to use [here](https://nebraltd.slack.com/archives/C02SVCWLHRB/p1642689522017200?thread_ts=1642680189.014100&cid=C02SVCWLHRB).

Appvales work [here](https://gitlab.com/appvales/nebra/-/commits/dev).

**Acceptance criteria**
- [x] [Text](https://github.com/NebraLtd/maker-starter-app/blob/6813eb34642b6c3bf5fa2bf1aeb61b54d0606cfb/src/locales/en.ts)
- [x] Update your theme colors in [theme.ts](src/theme/theme.ts). The theme controls the look of the application.

**Comment by Andrewsopoku** [comment]
@marvinmarnold  these are the screens that need immediate textual update 

1. Landing Screen
<img src="https://user-images.githubusercontent.com/20680743/156050218-aa40fc55-4d31-4a55-84e1-75188d663aa7.PNG" width="300" height="auto">

2. Menu/ Add Miner Screen
<img src="https://user-images.githubusercontent.com/20680743/156050568-876edb21-12d4-431b-9481-bdce61f5b6b4.PNG" width="300" height="auto">

3. Add Hotspot Screen
<img src="https://user-images.githubusercontent.com/20680743/156055605-bc35702e-7362-4172-a5dd-da699b002d1c.PNG" width="300" height="auto">




4. Bluetooth Setup Instruction Screens
<img src="https://user-images.githubusercontent.com/20680743/156051917-738bc68a-c210-4479-b458-ecafd1230ce7.PNG" width="300" height="auto">
<img src="https://user-images.githubusercontent.com/20680743/156052383-36944a81-8f33-40d6-9b91-b03078c9d3cc.jpeg" width="300" height="auto">



5. Scan QR code Screen
<img src="https://user-images.githubusercontent.com/20680743/156053102-e4d7b7c6-1cef-4d63-9115-36e0dc4c7711.PNG" width="300" height="auto">

**Comment by marvinmarnold** [comment]
@Andrewsopoku @posterzh, picking up the convo about splash screen. @posterzh could you detail the process you have in mind?

**Comment by posterzh** [comment]
In [expo-splash-screen documentation](https://www.npmjs.com/package/expo-splash-screen), it's clear said that we should place image assets in native android/ios project directory.

To change splash screen:
(shared assets)
• https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/src/assets/images/SplashScreen.png
• https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/src/assets/images/SplashScreen%402x.png
• https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/src/assets/images/SplashScreen%403x.png
(android)
• https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable/splashscreen_image.png
• https://github.com/NebraLtd/maker-starter-app/tree/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable-hdpi
• https://github.com/NebraLtd/maker-starter-app/tree/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable-mdpi
• https://github.com/NebraLtd/maker-starter-app/tree/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable-xhdpi
• https://github.com/NebraLtd/maker-starter-app/tree/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable-xxhdpi
• https://github.com/NebraLtd/maker-starter-app/tree/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable-xxxhdpi
(ios)
• https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/ios/MakerApp/Images.xcassets/SplashScreen.imageset/SplashScreen.png
• https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/ios/MakerApp/Images.xcassets/SplashScreen.imageset/SplashScreen%402x.png
• https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/ios/MakerApp/Images.xcassets/SplashScreen.imageset/SplashScreen%403x.png

**Comment by marvinmarnold** [comment]
@Louisreed an antenna [icon](https://github.com/NebraLtd/maker-starter-app/blob/081c8bd9230103aa1cfb2323f530fd1eca2abca2/src/makers/example/icon.svg) is needed can you help?
![image](https://user-images.githubusercontent.com/1311030/156481269-31252b2c-7a59-470c-ba17-c2f923709eaa.png)

**Comment by Andrewsopoku** [comment]
> In [expo-splash-screen documentation](https://www.npmjs.com/package/expo-splash-screen), it's clear said that we should place image assets in native android/ios project directory.
> 
> To change splash screen: (shared assets) • https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/src/assets/images/SplashScreen.png • https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/src/assets/images/SplashScreen%402x.png • https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/src/assets/images/SplashScreen%403x.png (android) • https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable/splashscreen_image.png • https://github.com/NebraLtd/maker-starter-app/tree/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable-hdpi • https://github.com/NebraLtd/maker-starter-app/tree/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable-mdpi • https://github.com/NebraLtd/maker-starter-app/tree/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable-xhdpi • https://github.com/NebraLtd/maker-starter-app/tree/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable-xxhdpi • https://github.com/NebraLtd/maker-starter-app/tree/2712ee44e09a2b9637ae80e18015f59186dc46d2/android/app/src/main/res/drawable-xxxhdpi (ios) • https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/ios/MakerApp/Images.xcassets/SplashScreen.imageset/SplashScreen.png • https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/ios/MakerApp/Images.xcassets/SplashScreen.imageset/SplashScreen%402x.png • https://github.com/NebraLtd/maker-starter-app/blob/2712ee44e09a2b9637ae80e18015f59186dc46d2/ios/MakerApp/Images.xcassets/SplashScreen.imageset/SplashScreen%403x.png

@posterzh  I understand you but my concern is that the manual approach is error prone and difficult as we have to generate different images for different definitions/dimensions.

Why not put the image in the native project and the CLI tool to do the necessary generation:

For icon or ic_launch change we can use:
`yarn icon`

For splash screen  change:
yarn expo-splash-screen  --background-color "<our preferred bg color>" --image-path ./src/assets/images/SplashScreen.png -r "cover"

**Comment by posterzh** [comment]
@Andrewsopoku I fully agree with you. As said in `expo-splash-screen` documentation, cli command works well for automating the splash image generation.

**Comment by Louisreed** [comment]
> @Louisreed an antenna [icon](https://github.com/NebraLtd/maker-starter-app/blob/081c8bd9230103aa1cfb2323f530fd1eca2abca2/src/makers/example/icon.svg) is needed can you help? ![image](https://user-images.githubusercontent.com/1311030/156481269-31252b2c-7a59-470c-ba17-c2f923709eaa.png)

How's this? Let me know if you need it in any other sizes/formats.

<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000"><defs><style>.cls-1{fill:#424242;}</style></defs><path class="cls-1" d="M519.5,757.5h-45a2,2,0,0,0-2,2v2h-7v-4h-25v4h-3v35h3v4h25v-4h7v2a2,2,0,0,0,2,2h45a2,2,0,0,0,2-2v-39A2,2,0,0,0,519.5,757.5Z"/><path class="cls-1" d="M510.5,306.5v-108h-2v-4h2V169a10.5,10.5,0,0,0-21,0v25.5h2v4h-2v108l-11,117v332a2,2,0,0,0,2,2h39a2,2,0,0,0,2-2v-332Z"/></svg>

**Comment by Louisreed** [comment]
`<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000"><defs><style>.cls-1{fill:#424242;}</style></defs><path class="cls-1" d="M519.5,757.5h-45a2,2,0,0,0-2,2v2h-7v-4h-25v4h-3v35h3v4h25v-4h7v2a2,2,0,0,0,2,2h45a2,2,0,0,0,2-2v-39A2,2,0,0,0,519.5,757.5Z"/><path class="cls-1" d="M510.5,306.5v-108h-2v-4h2V169a10.5,10.5,0,0,0-21,0v25.5h2v4h-2v108l-11,117v332a2,2,0,0,0,2,2h39a2,2,0,0,0,2-2v-332Z"/></svg>`

**Comment by marvinmarnold** [comment]
@Louisreed whoops I should have asked for a svg of a hotspot, not an antenna. Also note how small that antenna was looking
![signal-2022-03-07-200235](https://user-images.githubusercontent.com/1311030/157151455-f819e37d-105b-4ee9-98ae-58b6ac7fa471.png)
.

**Comment by Louisreed** [comment]
@marvinmarnold We don't have the digital files for the ROCKPi miner yet so I can't produce an SVG! Will a high-res PNG work for now?

**Comment by marvinmarnold** [comment]
@Louisreed could you provide SVGs for the original outdoor and indoor miners in that case? It's hard to tell if there will be any issues on different sized phones if we use a PNG.

**Comment by Louisreed** [comment]
@marvinmarnold 

![Nebra-Original-Indoor-Hotspot](https://user-images.githubusercontent.com/9106988/157424208-5f3a3bb2-010d-4205-88db-433bb2cabdce.svg)
![Nebra-Outdoor-Hotspot](https://user-images.githubusercontent.com/9106988/157424226-242d4eb9-228f-4311-8541-ee6d76160aab.svg)
![Nebra-ROCKPi-Indoor-Hotspot](https://user-images.githubusercontent.com/9106988/157424230-a559def4-e7bb-41c4-a88e-f5476b5b66f6.svg)

**Comment by marvinmarnold** [comment]
Awesome, thanks. @Louisreed I believe a zillion people have already talked to you about the color palette, but could you catch me up here. Are you doing any work currently to update [the colors](https://github.com/NebraLtd/maker-starter-app/blob/9af5c00858080dbec48b46ab55bd80d9a32fd2ea/src/theme/theme.ts#L79) in the app? Where's the best place to find our official color codes?


cc: @vladimirpoleshko

**Comment by Louisreed** [comment]
@marvinmarnold No problem! We're migrating everything to Figma, I've just seen and approved your request. I'll be updating this over the course of the next few weeks but for now everything you need should be in there!

**Comment by marvinmarnold** [comment]
@Louisreed we need icons to use on these four screens:
![signal-2022-03-14-112737_003](https://user-images.githubusercontent.com/1311030/158217700-18d3980a-94b7-43b5-a142-1be88609a98e.png)
![signal-2022-03-14-112737_004](https://user-images.githubusercontent.com/1311030/158217705-e6e41fc0-815a-4dee-ad60-ad61a6e8a710.png)
![signal-2022-03-14-112737_007](https://user-images.githubusercontent.com/1311030/158217707-a440339f-2f8e-4ab4-852f-20d07d282371.png)
![signal-2022-03-14-112737_008](https://user-images.githubusercontent.com/1311030/158217709-96651008-cc63-4da1-b288-d716ae2aa7ca.png)

**Comment by Louisreed** [comment]
@marvinmarnold Okay sure thing, I'll work on these next!

**Comment by Louisreed** [comment]
@marvinmarnold

![Nebra-Setup-Icon-1-Mesh](https://user-images.githubusercontent.com/9106988/158293122-486ef7f7-2651-4d12-8f46-64f72003053c.svg)
![Nebra-Setup-Icon-2-buildings](https://user-images.githubusercontent.com/9106988/158293124-22348f82-bf5c-4234-b0bc-83f1bac8ee00.svg)
![Nebra-Setup-Icon-3-view](https://user-images.githubusercontent.com/9106988/158293125-6769863a-a601-4368-9295-41d48aa07d03.svg)
![Nebra-Setup-Icon-4-window](https://user-images.githubusercontent.com/9106988/158293126-5be6976e-e6e2-4a11-ad67-73a9dcfe59f5.svg)

**Comment by Andrewsopoku** [comment]
Also should we not replace the "Welcome to Nebra" with Nebra logo? If so @Louisreed please can we get a better version of the logo below.
<img height="500" width="auto" src="https://user-images.githubusercontent.com/20680743/158542405-edbcda62-09b0-48cc-9ee1-d1b01bc73a94.PNG">

<img src="https://cdn.shopify.com/s/files/1/0071/2281/3001/files/Nebra-Icon-Blue_32x32.png?v=1610776627">

**Comment by marvinmarnold** [comment]
@Andrewsopoku will you take the lead replacing the logo? Then this ticket can be closed.

**Comment by Andrewsopoku** [comment]
@Louisreed please any update on the logo?

**Comment by Louisreed** [comment]
@Andrewsopoku Apologies I didn't get this notification! I'll upload the logo now!

**Comment by Louisreed** [comment]
<?xml version="1.0" encoding="utf-8"?>

<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 500 85.5" style="enable-background:new 0 0 500 85.5;" xml:space="preserve">
<style type="text/css">
	.st0{fill-rule:evenodd;clip-rule:evenodd;fill:#02A8F5;}
</style>
<g id="Nebra-Logo-Primary_00000157992477642921068340000011112989095797827263_">
	<g id="Nebra-Logo_00000047755535818278141780000003850824144845631140_">
		<g>
			<g>
				<path class="st0" d="M393.8,85.4h30.5l5.1-13.9h37.3l5.1,13.9h23.7c1.8-1.6,3.2-3.6,4.4-5.7L465.7,0h-34.8l-29.2,67.2
					l-14.8-23.9c7.7-1.9,13.6-8.1,13.6-19.5C400.6,9.6,392.9,0,372,0h-68.1c1.7,9,4,21.4,10.7,28.1c4.6,4.6,11.3,6.8,17.5,8.4V19.2
					h27.4c6.9,0,10.6,3.6,10.6,9.2c0,17.9-38.8,6.5-55.5,23.3c-7.1,7.1-11,24.2-11,33.8h28.5V57h26L376,85.4H393.8z M3.2,85.5h24.4
					v-54h0.2l40.9,54H97c1-1.7,3.1-5.8,3.1-7.7V7.8c0-1.9-2.1-6-3.1-7.6H72.6v53.5h-0.2L32.8,0.1H3.1C1.9,2,0.2,5.5,0,7.7v70
					C0,79.9,2.1,83.8,3.2,85.5L3.2,85.5z M111.9,85.4h79.3c1-1.6,3.1-5.7,3.1-7.6V67h-57.1V51.7h49.9V34h-49.9V18.4h57.1V7.6
					c0-1.9-2.1-6-3.1-7.6h-79.3c-1,1.8-3.1,5.7-3.1,7.8v70C108.7,79.7,110.8,83.8,111.9,85.4L111.9,85.4z M204.8,85.4
					c-1-1.6-3.1-5.7-3.1-7.6V7.6c0-1.9,2.1-6,3.1-7.6h67.3c17.3,0,28,12.9,22.5,23.7c-6.7,11.1-21.5,13.3-33.2,15.4
					c-0.4,0-0.7,0.4-0.7,0.8c0,0.3,0.3,0.6,0.7,0.7c6.1,0.9,12.2,2.4,18.1,4.3c27.2,9.3,26,40.5-9.6,40.5L204.8,85.4z M266.9,25.6
					c0-4.9-4-7.1-8.5-7.2H230v14.5h28.1C264.2,32.9,266.9,29.5,266.9,25.6L266.9,25.6z M270.5,58.1c0-5.8-4.4-8.5-9.8-8.5H230V67
					h30.4C266.4,67,270.5,63.8,270.5,58.1L270.5,58.1z M460.1,53.8h-24L448,21.5h0.2L460.1,53.8z"/>
			</g>
		</g>
	</g>
</g>
</svg>

**Comment by Louisreed** [comment]
```
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 26.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 500 85.5" style="enable-background:new 0 0 500 85.5;" xml:space="preserve">
<style type="text/css">
	.st0{fill-rule:evenodd;clip-rule:evenodd;fill:#02A8F5;}
</style>
<g id="Nebra-Logo-Primary_00000157992477642921068340000011112989095797827263_">
	<g id="Nebra-Logo_00000047755535818278141780000003850824144845631140_">
		<g>
			<g>
				<path class="st0" d="M393.8,85.4h30.5l5.1-13.9h37.3l5.1,13.9h23.7c1.8-1.6,3.2-3.6,4.4-5.7L465.7,0h-34.8l-29.2,67.2
					l-14.8-23.9c7.7-1.9,13.6-8.1,13.6-19.5C400.6,9.6,392.9,0,372,0h-68.1c1.7,9,4,21.4,10.7,28.1c4.6,4.6,11.3,6.8,17.5,8.4V19.2
					h27.4c6.9,0,10.6,3.6,10.6,9.2c0,17.9-38.8,6.5-55.5,23.3c-7.1,7.1-11,24.2-11,33.8h28.5V57h26L376,85.4H393.8z M3.2,85.5h24.4
					v-54h0.2l40.9,54H97c1-1.7,3.1-5.8,3.1-7.7V7.8c0-1.9-2.1-6-3.1-7.6H72.6v53.5h-0.2L32.8,0.1H3.1C1.9,2,0.2,5.5,0,7.7v70
					C0,79.9,2.1,83.8,3.2,85.5L3.2,85.5z M111.9,85.4h79.3c1-1.6,3.1-5.7,3.1-7.6V67h-57.1V51.7h49.9V34h-49.9V18.4h57.1V7.6
					c0-1.9-2.1-6-3.1-7.6h-79.3c-1,1.8-3.1,5.7-3.1,7.8v70C108.7,79.7,110.8,83.8,111.9,85.4L111.9,85.4z M204.8,85.4
					c-1-1.6-3.1-5.7-3.1-7.6V7.6c0-1.9,2.1-6,3.1-7.6h67.3c17.3,0,28,12.9,22.5,23.7c-6.7,11.1-21.5,13.3-33.2,15.4
					c-0.4,0-0.7,0.4-0.7,0.8c0,0.3,0.3,0.6,0.7,0.7c6.1,0.9,12.2,2.4,18.1,4.3c27.2,9.3,26,40.5-9.6,40.5L204.8,85.4z M266.9,25.6
					c0-4.9-4-7.1-8.5-7.2H230v14.5h28.1C264.2,32.9,266.9,29.5,266.9,25.6L266.9,25.6z M270.5,58.1c0-5.8-4.4-8.5-9.8-8.5H230V67
					h30.4C266.4,67,270.5,63.8,270.5,58.1L270.5,58.1z M460.1,53.8h-24L448,21.5h0.2L460.1,53.8z"/>
			</g>
		</g>
	</g>
</g>
</svg>

```

**Comment by marvinmarnold** [comment]
This was closed by #69
